.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,103 @@
+---
+name: E2E Tests
```

Comment:
I see we renaming the status name from `integration` -> `e2e tests`

![Screenshot 2023-01-04 at 22-56-26 CI Use Ansible Molecule for end-to-end testing (Rework) by mamercad · Pull Request #329 · StackStorm_ansible-st2](https://user-images.githubusercontent.com/1533818/210665580-ca6e4774-5a71-475c-8fe0-ab08ede65849.png)

Or we rename the [`tests/integration`](https://github.com/StackStorm/ansible-st2/tree/master/test/integration) dir or the `e2e tests` workflow.


I'm fine either way, as long as it's consistent in both places.

Reply:
As mentioned previously, once we turn this repository into a collection, the `tests/integration` directory will be needed, see [this](https://docs.ansible.com/ansible/latest/dev_guide/developing_collections_structure.html) for the structure of a collection. Though, the format of what's inside the `tests/integration` directory will change a bit, and, we'll want to start using `ansible-test` at that point in time (which doesn't necessarily obviate Molecule).

Reply:
I'm not sure what you mean by "consistent in both places", can you clarify?